### PR TITLE
Minor updates to the writing a query page

### PR DESCRIPTION
**Pull request by susodapop**
* Fixed heading sizes so side menu renders nicely
* Added info about experimental manage permissions support

### CODE REVIEW

File: src/pages/kb/user-guide/querying/writing-queries.md [hunk 1]
```diff
@@ -60,18 +63,27 @@ By default each query starts as an unpublished draft, which means that:
...
-### Duplicating (Forking) a Query
+## Duplicating (Forking) a Query
 
 If you need to create a copy of an existing query (created by you or someone else), you can fork it. To fork a query, just click on the Fork button (see example below)
 
 ![](/assets/images/docs/gifs/queries/fork_query.gif)
+
+## Managing Query Permissions
+
+By default, saved queries can only be modified by the user who created them and members of the Admin group. But Redash includes experimental support to share edit permissions with non-Admin users. You need to enable it first. Open your organization settings and check the "Enable experimental multiple owners support"
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/writing-queries.md]: Worth mentioning that only an admin can enable this feature.

File: src/pages/kb/user-guide/querying/writing-queries.md [hunk 2]
```diff
@@ -60,18 +63,27 @@ By default each query starts as an unpublished draft, which means that:
...
 ![](/assets/images/docs/gitbook/archive_query.png)
 
-### Duplicating (Forking) a Query
+## Duplicating (Forking) a Query
 
 If you need to create a copy of an existing query (created by you or someone else), you can fork it. To fork a query, just click on the Fork button (see example below)
 
 ![](/assets/images/docs/gifs/queries/fork_query.gif)
+
+## Managing Query Permissions
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/writing-queries.md]: This is applicable for dashboards as well. Maybe it needs its own home? :)

Comment (susodapop) [reply src/pages/kb/user-guide/querying/writing-queries.md]: For the moment I've just copied the text to the dashboard page. Long term, what would you think about putting it under the users, groups, and permissions section?

File: src/pages/kb/user-guide/querying/writing-queries.md (hunk 1 above)
Comment (susodapop) [reply src/pages/kb/user-guide/querying/writing-queries.md]: Fixed.

File: src/pages/kb/user-guide/querying/writing-queries.md (hunk 2 above)
Comment (arikfr) [reply src/pages/kb/user-guide/querying/writing-queries.md]: Makes sense. And maybe link to it from the relevant query/dashboard pages.

**Pull request merged**